### PR TITLE
Fix Issue #412: Rename NSColor to Color in ColorExtensions.swift

### LINKED ISSUE #412: Rename NSColor to Color in ColorExtensions.swift
I noticed today [ColorExtensions.swift](https://github.com/SwifterSwift/SwifterSwift/blob/master/Sources/Extensions/Shared/ColorExtensions.swift) has some places where we use `NSColor` or `UIColor` in the docs instead of `Color`.

We want to keep using both `UIColor` and `NSColor`  **ONLY** in examples to let users know that extensions work for both types, and replace them with `Color` everywhere else.

### PR DESCRIPTION
**Pull request by DannyMac180**
Renamed NSColor and UIColor to Color

Resolves #412 

## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 4.
- [ ] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [ ] I have added tests for new extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [ ] I have added a changelog entry describing my changes.

### CODE REVIEW

File: Sources/Extensions/Shared/ColorExtensions.swift [hunk 1]
```diff
@@ -31,9 +31,9 @@ public extension Color {
 	
 	/// SwifterSwift: RGB components for a Color (between 0 and 255).
 	///
-	///		UIColor.red.rgbComponents.red -> 255
-	///		NSColor.green.rgbComponents.green -> 255
-	///		UIColor.blue.rgbComponents.blue -> 255
+	///		Color.red.rgbComponents.red -> 255
```
Comment (SD10) [inline Sources/Extensions/Shared/ColorExtensions.swift]: I think the issue states that we should not fix the names in the example blocks. Only in the actual documentation

File: Sources/Extensions/Shared/ColorExtensions.swift [hunk 2]
```diff
@@ -255,7 +255,7 @@ public extension Color {
 // MARK: - Initializers
 public extension Color {
 	
-	/// SwifterSwift: Create NSColor from RGB values with optional transparency.
+	/// SwifterSwift: Create Color from RGB values with optional transparency.
```
Comment (SD10) [inline Sources/Extensions/Shared/ColorExtensions.swift]: So basically this is the change we're looking for

Comment (DannyMac180) [reply Sources/Extensions/Shared/ColorExtensions.swift]: @SD10 thanks for the clarification. I'll go back and adjust this and submit a new pull request.

Comment (SD10) [reply Sources/Extensions/Shared/ColorExtensions.swift]: @DannyMac180 No need for a new pull request. You can just make another commit changing them

Comment (DannyMac180) [reply Sources/Extensions/Shared/ColorExtensions.swift]: @SD10 thanks for your help! Maybe you can tell, this is my first attempt at contributing to an open source project.

Comment (SD10) [reply Sources/Extensions/Shared/ColorExtensions.swift]: No problem @DannyMac180. You'll be a pro in no time 😁 👍 Thank you for submitting this fix 💯

**Comment by SD10** [comment]
Thank you for contributing to SwifterSwift! I've invited you to join the SwifterSwift GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md). Feel free to reach out if you have any questions! 😃

**Pull request merged**